src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpGen.cs

```diff
@@ -54,6 +56,40 @@ public async Task<GeneratedCodeWorkspace> ExecuteAsync(CodeModel codeModel)
             return project;
         }
 
+        private async Task<CSharpCompilation?> LoadBaselineContract()
```

Comment:
I think maybe we could put this method inside the `SourceInputModel` class so that we do not have to manually enable this for all targets (for instance we have to add a method call on those consuming tsp)

Reply:
Updated